### ISSUE TITLE
Alpha 1.6 Ents

### ISSUE THREAD

**Issue by Ladysarajane** [teagan75/Witchery#44]
I have been after Ent to try to get a twig before my difficulty level went up to medium. I have to make more sleeping potion for the spirit world since I keep having bad luck and keep getting nightmares instead of dreams with the dreamcatcher.  (I was in dream world and I got mobbed by silverfish). I need more whiff of magic to make the potion, so I figured to get two birds with one stone  and started cutting down the witchery trees, like one after another. I grew more  and chopped them.  I even accidentally set off my trapped poppy flowers (I love those!!!!) I set out to slow ents, I had cut 15 trees and NOT one ENT spawned. Not sure if something changed when you messed with creepers, but I found it odd, That I would cut that many magic trees at one time, and not have one spawn.

**Comment by teagan75** [comment]
That is strange and hopefully just really bad luck. The mod I use is very
specific as I have to write the mob name in the config. Currently only
creepers and enderman are disabled.

**Comment by Ladysarajane** [comment]
I will keep trying and report. Usually if I cut 3-5 witchery trees in a row, An ent or two spawns, but this time nothing. I will keep trying. I can always use the saplings for brews.

**Comment by teagan75** [comment]
are you still on peaceful? just thought I would ask that

**Comment by Ladysarajane** [comment]
No, this playthrough I have been keeping it untouched.. This is a new world with a new files. I have been getting wolves and zombies and skeletons and spiders.  I thought that too and doubled checked. The setting is normal spawns.

**Comment by satanKami** [comment]
i got ents before... but yes they become really rare when it's hard

**Comment by Ladysarajane** [comment]
But I am not at hard, they should have been spawning at the easy level. I even tried a little right after I got mutandis and made the trees. 
After about 3 stacks of bone meal and basically spamming the witchery trees, I finally got an ent and an ent twig. It didn't used to be this hard to get ents to spawn. Especially after spamming them and cutting them one after another. Well, since I did get one to spam, I can close this issue. They are just hard to get to spawn.
